src/main/java/org/sonarsource/scanner/cli/Conf.java

```diff
@@ -70,15 +69,6 @@ Properties properties() {
     return result;
   }
 
-  boolean isSonarCloud(@Nullable Properties testProperties) {
-    String hostUrl = testProperties != null ? testProperties.getProperty(PROPERTY_SONAR_HOST_URL) : properties().getProperty(PROPERTY_SONAR_HOST_URL);
```

Comment:
This line was bad, because calling `properties()` method would load configuration again from files.